tsdb/head.go

```diff
@@ -424,6 +438,31 @@ func NewHead(r prometheus.Registerer, l log.Logger, wal *wal.WAL, opts *HeadOpti
 
 func mmappedChunksDir(dir string) string { return filepath.Join(dir, "chunks_head") }
 
+func (h *Head) ApplyConfig(cfg *config.Config) error {
+	if h.opts.EnableExemplarStorage && cfg.StorageConfig.ExemplarsConfig.MaxExemplars > -1 {
```

Comment:
We could use this to reduce indentation for the entire function
```suggestion
	if !h.opts.EnableExemplarStorage || cfg.StorageConfig.ExemplarsConfig.MaxExemplars < 0 {
```